bcipy/helpers/task.py

```diff
@@ -414,32 +411,29 @@ def __call__(self,
         timing_info = tmp_timing
 
         n_inquiry = len(timing_info) // trials_per_inquiry
+        trial_duration_samples = int(trial_length * fs)
 
         # triggers in seconds are mapped to triggers in number of samples.
         triggers = list(map(lambda x: int((x + offset) * fs), timing_info))
 
-        # shape (Channels, Inquiries, Samples)
-        reshaped_data = np.zeros((len(eeg_data), n_inquiry, num_samples))
-
         # Label for every inquiry
-        labels = np.zeros(n_inquiry)
-        for inquiry_idx, chunk in enumerate(grouper(zip(trial_labels, triggers), trials_per_inquiry)):
+        labels = np.zeros(n_inquiry, dtype=np.long)
```

Comment:
Would you mind doing this with the `TrialReshaper` in this PR too?

Reply:
Sure